lib/tools/adb-commands.js

```diff
@@ -35,6 +35,15 @@ methods.getApiLevel = async function () {
   }
 };
 
+methods.getPlatformVersion = async function () {
+  log.info("Getting device platform version");
+  try {
+    return this.shell(['getprop', 'ro.build.version.release']);
```

Comment:
our new style guide says this should be: `return await this.shell(['getprop', 'ro.build.version.release']);`